### ISSUE TITLE
Leech

### ISSUE THREAD

**Issue by ToshiroZ** [springmusk026/Imgui-Unity#5]
Who gave you the code for hooking the jni functions
I was the one who wrote that code and I am wondering who leaked it to you

**Comment by springmusk026** [comment]
lol , another bastard  get lost from here

camaining the source lol , i neither know you nor this source have any code of your so stop faking the world

**Comment by ToshiroZ** [comment]
Except it does have my code 
The hook you have for nativeInputEvent was created by me

**Comment by ToshiroZ** [comment]
I had to go through some games in unity to find ways to handle input and I came across it from the small
I then had to find a way to find the address to that function.
I checked the exports and noticed it wasn't in the exports so I decided to hook onto RegisterNatives to see if it was being registered through it and it was.
There are also a few other methods that are registered from there. Mind proving you actually wrote any of this and telling me even one of those other methods registered?
Also, no java is required for that menu to function if you did the ImGui implementation by hooking swapbuffers.
I know this as I was one of the first to get a fully working ImGui template working on android. I also made it from scratch without skidding and relying on Lgl's template which is a shit paste. Also switch to cmake
